src/java.desktop/share/classes/sun/awt/AppContext.java

```diff
@@ -104,20 +104,14 @@
  * Since a separate AppContext can exist for each ThreadGroup, trusted
  * and untrusted code have access to different Foo instances.  This allows
  * untrusted code access to "system-wide" services -- the service remains
- * within the AppContext "sandbox".  For example, say a malicious applet
+ * within the AppContext "sandbox".  For example, say malicious code
  * wants to peek all of the key events on the EventQueue to listen for
  * passwords; if separate EventQueues are used for each ThreadGroup
- * using AppContexts, the only key events that applet will be able to
- * listen to are its own.  A more reasonable applet request would be to
+ * using AppContexts, the only key events that code will be able to
+ * listen to are its own.  A more reasonable request would be to
  * change the Swing default look-and-feel; with that default stored in
- * an AppContext, the applet's look-and-feel will change without
- * disrupting other applets or potentially the browser itself.<p>
- *
- * Because the AppContext is a facility for safely extending application
- * service support to applets, none of its methods may be blocked by a
- * a SecurityManager check in a valid Java implementation.  Applets may
- * therefore safely invoke any of its methods without worry of being
- * blocked.
+ * an AppContext, the look-and-feel will change without
+ * disrupting other contexts.<p>
```

Comment:
```suggestion
 * disrupting other contexts.
```
The empty paragraph can be dropped.

Reply:
ok